chart/templates/openvsx-proxy-configmap.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
 data:
   config.json: |
     {
-      "log_debug": true,
```

Comment:
Should we update installer?

Reply:
Oh man, yeah right. I still forget about the installer. :roll_eyes:

Reply:
https://github.com/gitpod-io/gitpod/pull/6781